README.md

```diff
@@ -192,7 +192,8 @@ and `{}` for a mutually exclusive keyword.
 | Switch database | `USE <database> [ROLE <role>];` | The role you set is used for accessing with [fine-grained access control](https://cloud.google.com/spanner/docs/fgac-about). |
 | Create database | `CREATE DATABSE <database>;` | |
 | Drop database | `DROP DATABASE <database>;` | |
-| List tables | `SHOW TABLES;` | |
+| List tables in a default schema| `SHOW TABLES;` | |
+| List tables in a named schema | `SHOW TABLES <schema>;` | |
```

Comment:
Maybe we can combine two rows into one like this.

```
| List tables in a schema | `SHOW TABLES [<schema>];` | If schema is not provided, default schema is used |
```

Reply:
Fixed in https://github.com/cloudspannerecosystem/spanner-cli/pull/173/commits/0d7393de10bf1a597b0276803a03ec9da1761dca